tests/units/iop/test_iop_parser.py

```diff
@@ -471,3 +591,9 @@ def test_get_license(parsed_articles):
     ]
     for license, article in zip(licenses, parsed_articles):
         assert article["license"] == license
+
+
+def test_get_collaborations(parsed_articles):
```

Comment:
This comment also is related for the other tests. Why we are adding `parsed_articles` fixture since we have to iterate. We can use the the `params` from `pytest` to have our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and without extra logic